e2e_test/udf/sql_udf.slt

```diff
@@ -326,6 +318,20 @@ create function type_mismatch(INT) returns varchar language sql as 'select $1 +
 statement error Expected an expression:, found: EOF at the end
 create function add_error(INT, INT) returns int language sql as $$select $1 + $2 +$$;
 
+######################################################################
+# Not yet supported features 🤪 (and potential basic use case tests) #
+######################################################################
+
+# 1. Recursion Support for SQL UDF
+
+# The execution will eventually exceed the pre-defined max stack depth
+# statement error function fib calling stack depth limit exceeded
+# select fib(100);
+
+# Currently create a materialized view with a recursive sql udf will be rejected
+# statement error function fib calling stack depth limit exceeded
+# create materialized view foo_mv as select fib(100);
```

Comment:
Why it is forbidden in mviews? I think there is no difference between batch query and mview? 🤔

Reply:
Since we currently just don't allow recursive sql udf no matter batch query or mview 🥵

Reply:
Oh I see. 😄